src/media/css/site.styl

```diff
@@ -284,7 +284,11 @@ a:active {
     }
     section.full > div {
         padding: 30px 0;
-        width: $max-site-width;
+        max-width: $max-site-width;
+    }
+    .placeholder + .placeholder section.full,
```

Comment:
Tried it. That doesn't work very well.

![screen shot 2015-02-18 at 12 16 17 pm](https://cloud.githubusercontent.com/assets/23885/6254713/ff5c4478-b767-11e4-9b4b-edfbb3d4245a.png)